### PR TITLE
feat: /iot redesign

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied changes as per [doc](https://docs.google.com/document/d/1nBx1SP3Y10gXX0rZPDaz1Gudy-hWP4XRMUcETyPBldI/edit?tab=t.0#heading=h.kbutqxr7kac7) and [design](https://www.figma.com/design/1CdIT6TV3ASf55GsnQcwXg/Internet-of-Things?node-id=1-9092&node-type=instance&m=dev)

## QA

- View the site locally in your web browser at: https://ubuntu-com-14537.demos.haus/iot
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-16628

### CODE REVIEW

**Comment by Sophie-32** [comment]
Responded to your comment in Figma with one change.

**Comment by mattea-turic** [comment]
thanks @mtruj013 :) adding a few comments:

For this section:
Screenshot 2024-12-04 at 10 05 39
- [ ]  I added an updated asset [here](https://assets.ubuntu.com/v1/7b0d858b-hero-img-iot-2.png) that reflects the design

For this section:
Screenshot 2024-12-04 at 10 17 17
- [ ] could the ctas be aligned pls (it looks good at ~1220 and up, but below they change position)
- [ ] and for smaller screens, could you wrap the paragraph in the "ubuntu core" card with a shallow, like you did for "ubuntu"

For this section:
Screenshot 2024-12-04 at 10 10 44
- [ ]  the third item should have heading "Dedicated Snap Store" and then "Develop your applications privately in an enterprise Store"
- [ ] could you also apply h5 styling to the subheadings with the muted colour pls?

For this section:
Screenshot 2024-12-04 at 10 13 34
- [ ] CTA should be "Let's discuss your business and help you find the right product"

**Comment by mattea-turic** [comment]
looks great, thanks @mtruj013 ! I just have one tiny thing to ask, is if you could replace the existing arm logo that's in the logo section with [this one](https://assets.ubuntu.com/v1/fdbc1be5-arm-logo-no-cutoff.png), as the current one is cut off a little on the side

I'll design +1 !

File: templates/internet-of-things/index.html [hunk 1]
```diff
@@ -10,578 +10,619 @@
...
+              </div>
+
+              <div class="p-equal-height-row__item">
+                <h3 class="p-heading--5">
+                  <a href="/internet-of-things/digital-signage">Digital signage</a>
+                </h3>
+              </div>
+
+              <p class="p-equal-height-row__item">
+                A small footprint and full OpenGL make Ubuntu Core the perfect platform for secure digital signs. Try Ubuntu Frame, a reliable and secure display server for embedded Linux devices with long term support.
```
Comment (petesfrench) [inline templates/internet-of-things/index.html:160]:
```suggestion
                A small footprint and full OpenGL make Ubuntu Core the perfect platform for secure digital signs designed with security in mind. Try Ubuntu Frame, a display server for embedded Linux devices with long term support.
```

File: templates/internet-of-things/index.html [hunk 2]
```diff
@@ -10,578 +10,619 @@
...
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Trusted by</h2>
+      </div>
+      <div class="col">
+        <div class="p-logo-section">
+          <div class="p-logo-section__items u-no-margin--bottom">
+            <div class="p-logo-section__item">
```
Comment (petesfrench) [inline templates/internet-of-things/index.html:279]: In the copydoc these logos are also links

File: templates/internet-of-things/index.html [hunk 3]
```diff
@@ -10,578 +10,619 @@
...
+      </div>
+    </div>
+    <div class="p-equal-height-row">
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <h3 class="p-heading--5 u-no-margin--bottom">Smart start</h3>
+          <p class="u-text--muted u-no-padding--top p-heading--5">Ensure the success of your application packaging on Ubuntu</p>
+        </div>
+        <p class="p-equal-height-row__item">
           Enjoy Snapcraft 101 training, snap store services, device fees and support for one year for up to 1,000 devices and much more as part of our smart start service. Find out more about <a href="/pricing/devices">what's included here</a>.
```
Comment (petesfrench) [inline templates/internet-of-things/index.html:404]:
```suggestion
          Enjoy Snapcraft 101 training, snap store services, device fees and support for one year for up to 1,000 devices and much more as part of our smart start service. Learn <a href="/pricing/devices">what's included with Smart Start</a>.
```

File: templates/internet-of-things/index.html [hunk 4]
```diff
@@ -10,578 +10,619 @@
...
-        <h3>Device Enablement</h3>
-        <p class="p-heading--4">Your own production-grade image in our testing infrastructure</p>
-        <p>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <h3 class="p-heading--5 u-no-margin--bottom">Device Enablement</h3>
+          <p class="u-text--muted u-no-padding--top p-heading--5">Your own production-grade image in our testing infrastructure</p>
+        </div>
+        <p class="p-equal-height-row__item">
           <a href="/core/services">Our Enablement services</a> allow you to get a production-grade Ubuntu image for your device, while our Maintenance services guarantee that any security update won't break your device.
```
Comment (petesfrench) [inline templates/internet-of-things/index.html:413]:
```suggestion
          Our<a href="/core/services"> Enablement services</a> allow you to get a production-grade Ubuntu image for your device, while our Maintenance services guarantee that any security update won't break your device.
```

File: templates/internet-of-things/index.html (hunk 2 above)
Comment (petesfrench) [reply templates/internet-of-things/index.html:279]: And missing alt texts on these logos

**Review by petesfrench — APPROVED** [review]
LGTM, just a couple content changes

**Pull request merged**